web-app/js/portal/data/DataCollection.js

```diff
@@ -46,15 +46,23 @@ Portal.data.DataCollection = function() {
     constructor.prototype.getFiltersRequestParams = function() {
         var layer = this.getLayerSelectionModel().getSelectedLayer();
         var serverType = layer.server.type.toLowerCase();
-        var layerName = serverType == 'geoservercore' ? layer.wmsName : this._getDownloadLayerName();
+        var filterLayer = this._getFilterLayer(serverType, layer);
 
         return {
             server: layer.url,
             serverType: serverType,
-            layer: layerName
+            layer: filterLayer
         };
     };
 
+    constructor.prototype._getFilterLayer = function(serverType, layer) {
```

Comment:
This is actually the layerName not the layer

Reply:
Renamed to filterLookupName